docs/pages/versions/unversioned/sdk/firebase-recaptcha.md

```diff
@@ -256,6 +256,7 @@ Modal screen that is automatically shown and displays a reCAPTCHA widget. The re
 
 - **firebaseConfig (IFirebaseOptions)** -- Firebase web configuration.
 - **firebaseVersion (string)** -- Optional version of the Firebase JavaScript SDK to load in the web-view. You can use this to load a custom or newer version. For example `version="6.8.0"`.
+- **appVerificationDisabledForTesting (boolean)** -- When set, this property disables app verification for the purpose of testing phone authentication. When this is disabled, a mock reCAPTCHA is rendered instead. This is useful for manual testing during development or for automated integration tests.
```

Comment:
```suggestion
- **appVerificationDisabledForTesting (boolean)** -- When set, this property disables app verification for the purpose of testing phone authentication. When this prop is `true`, a mock reCAPTCHA is rendered. This is useful for manual testing during development or for automated integration tests.
```

maybe good to link to the firebase docs for this prop as well for more info

Reply:
Thanks, I've added a link to firebase as well 👍